src/src/main/java/utils/IdGenerator.java

```diff
@@ -0,0 +1,7 @@
+package utils;
+
+public class IdGenerator {
+    public static long generateId(){
+        return (long) (Math.random() * 1000000);
```

Comment:
we need increment id all time! from 0 to ....

Reply:
fixed